lib/components/primitive-components/Group/Subcircuit/inflators/inflatePcbTrace.ts

```diff
@@ -0,0 +1,22 @@
+import { PcbTrace } from "lib/components/primitive-components/PcbTrace"
+import type { InflatorContext } from "../InflatorFn"
+import type { PcbTrace as PcbTraceElement } from "circuit-json"
+
+/**
+ * Preserve routed PCB traces from circuitJson by seeding the subcircuit's
+ * pcbRouteCache. This lets trace rendering reuse existing routes instead of
+ * rerouting from scratch.
+ */
+export const inflatePcbTrace = (
+  pcbTrace: PcbTraceElement,
+  inflatorContext: InflatorContext,
+) => {
+  const { subcircuit } = inflatorContext
+
+  subcircuit.add(
+    new PcbTrace({
+      route: pcbTrace.route,
+      source_trace_id: pcbTrace.source_trace_id,
```

Comment:
this isn't right- you need to create a `Trace` not a `PcbTrace`, and you can't use the source_trace_id of the original circuitJson (that wouldn't work at all, there is nothing to create that source trace)